nav2_docking/opennav_docking/src/simple_charging_dock.cpp

```diff
@@ -284,19 +284,23 @@ void SimpleChargingDock::jointStateCallback(const sensor_msgs::msg::JointState::
 {
   double velocity = 0.0;
   double effort = 0.0;
+  size_t match_count = 0;  
   for (size_t i = 0; i < state->name.size(); ++i) {
     for (auto & name : stall_joint_names_) {
       if (state->name[i] == name) {
         // Tracking this joint
         velocity += abs(state->velocity[i]);
         effort += abs(state->effort[i]);
+        ++match_count;
       }
     }
   }
 
   // Take average
-  effort /= stall_joint_names_.size();
-  velocity /= stall_joint_names_.size();
+  if (match_count > 0) {
```

Comment:
Is this just to avoid a divide by zero issue? If so, we could just have a `bool` instead which would be more intuitive unless we're going to use the `count` rather than the `size` (or check that `count == size` or something to check we have all the stuff needed)

Reply:
If this is really targeting a divide by zero, we probably should update the configure() to not create the subscriber if stall_joint_names_ is empty() - it already prints an error, but there is no reason to have the subscriber run the callback if there are no names to match